### PR TITLE
Shuffle player starting position

### LINKED ISSUE #8: Shuffle player starting position as an option

### PR DESCRIPTION
**Pull request by joseph-passineau**
Added the shufflePlayers option in order to shuffle starting positions.

closes #8

### CODE REVIEW

File: src/client/utils.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import fs from 'fs';
 
 import { DEFAULT_SERVER_PORT } from '../constants';
 import { IGameState, IJoinRoomOpts, ISimpleGameState } from '../types';
+import { boolean } from '@colyseus/schema/lib/encoding/decode';
```
Comment (asimmon) [inline src/client/utils.ts]: I think that this import has been automatically added by the IDE and it is not necessary

File: src/client/utils.ts [hunk 2]
```diff
@@ -45,6 +47,10 @@ export function getJoinOptions(): IJoinRoomOpts {
     throw new Error('Invalid option --mode, values are training, match or spectate');
   }
 
+  if (typeof config.shufflePlayers === 'boolean') {
```
Comment (asimmon) [inline src/client/utils.ts]: We could also add shufflePlayers: false by default in config.json

Comment (joseph-passineau) [reply src/client/utils.ts]: We could, but I feel this is better if its an optional config. We should have the config file only contain crucial settings.

File: src/client/utils.ts (hunk 1 above)
Comment (joseph-passineau) [reply src/client/utils.ts]: Removed

**Pull request merged**